conans/client/remover.py

```diff
@@ -76,7 +76,7 @@ def remove_packages(self, package_layout, ids_filter=None):
                 if not package_layout.package_exists(pref):
                     raise PackageNotFoundException(pref)
                 pkg_folder = package_layout.package(pref)
-                self._remove(pkg_folder, package_layout.ref, "package:%s" % package_id)
+                package_layout.package_remove(pref)
                 self._remove_file(pkg_folder + ".dirty", package_layout.ref, "dirty flag")
                 self._remove_file(package_layout.system_reqs_package(pref), package_layout.ref,
                                   "%s/%s" % (package_id, SYSTEM_REQS))
```

Comment:
Is this something we can move into the `package_remove` function? I would say that I can remove always the `.dirty` check and the system-requires associated with a package-ID if I remove the package-ID from the cache, right?